include/m_npc_schedule.h

```diff
@@ -0,0 +1,44 @@
+#ifndef M_NPC_SCHEDULE_H
+#define M_NPC_SCHEDULE_H
+
+#include "ultra64.h"
+#include "m_npc.h"
+
+typedef struct NpsScheduleData {
+    /* 0x00 */ u32 type;
```

Comment:
Is this type a `NpsScheduleType`?

Reply:
Yeah it is, I didn't realise

Reply:
Would using `NpsScheduleType` instead of `u32` match?

Reply:
It does match, so I'll push a change